requests_toolbelt/utils/dump.py

```diff
@@ -76,11 +76,11 @@ def _dump_request_data(request, prefixes, bytearr, proxy_info=None):
     bytearr.extend(prefix + b'\r\n')
     if request.body:
         if isinstance(request.body, compat.basestring):
-            bytearr.extend(prefix + _coerce_to_bytes(request.body))
+            bytearr.extend(prefix + _coerce_to_bytes(request.body) + b'\r\n')
         else:
             # In the event that the body is a file-like object, let's not try
             # to read everything into memory.
-            bytearr.extend(b'<< Request body is not a string-like type >>')
+            bytearr.extend(b'<< Request body is not a string-like type >>' + b'\r\n')
```

Comment:
We don't need string concatenation here, just add `\r\n` after `>>` and before `'`

Reply:
You're right. I'll fix this